catalog/video/catalog-mpegts-stream/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -52,6 +52,11 @@
                 name="Location Subsample Count" id="klvLocationSubsampleCount" required="true"
                 type="Integer" default="50"/>
 
+        <AD
+                description="Delay updates to the parent stream metacard and child video chunk metacard to give the CatalogProvider time to create new metacards. A longer delay reduces the likelihood of retries when updating. (seconds)"
```

Comment:
Can you dumb this down to an admin who has no idea how the system works?
Something like: `Delay update frequency. This sets how often updates to the catalog occur.`